scss/donate.scss

```diff
@@ -1,36 +1,173 @@
 /*@use 'vars';*/
 //temporary until resolve _vars.scss issue
-$cyan: #82ecec;
-$pink: #f27aaa;
-$lavender: #b5a1c4;
-$light-lavender: #cca8c1;
-$ex-light-lavender: #e9e3ed;
-$hero-blue: #6fcbdc;
 
-.donate-button {
-  text-align: center;
-  color: #fff;
-  border: none;
-  background-color: #ff5789;
-  align-items: center;
-  padding: 7px 35px;
-  font-size: 35px;
-  font-weight: bold;
-  border-radius: 6px;
+HERO SECTION BEGINS HERE
+.hero-clouds {
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  z-index: 1;
+  @media only screen and (max-width: 767px) {
+    width: 834px;
+    height: 663px;
+    left: -100px;
+    top: -26px;
+  }
+
+  @media only screen and (max-width: 1024px) {
+    width: 834px;
+    height: 663px;
+    left: 0px;
+    top: 69px;
+  }
 }
 
-.donate-button:hover {
-  opacity: 75%;
-  text-decoration: none;
-  color: #fff;
-  transition: 0.3s ease;
+.hero {
+  position: relative;
+  overflow: hidden;
```

Comment:
"donating to" above the technica logo disappears on mobile, `overflow: hidden` is likely the culprit, delete
<img width="500" alt="Screen Shot 2021-06-24 at 12 13 54 AM" src="https://user-images.githubusercontent.com/62481939/123201733-477a5000-d481-11eb-8cd2-1e8f1440177d.png">